mongodb/invites.go

```diff
@@ -1,4 +1,4 @@
-package collections
+package mongodb
```

Comment:
There's now a couple threaddb collections. So, I moved all the mongo stuff to `mongodb` and the threads stuff now lives in `threaddb`: `buckets` and `mail` are the two built-in thread collections.